README.md

```diff
@@ -132,6 +136,12 @@ or specifying the database name on the command-line:
 cat animaldb.txt | couchrestore --db animaldb2
 ```
 
+### Compatibility note
+
+Newer versions of couchbackup can restore backups created by older versions within the same major version.
+
+Do not use an older version of couchbackup to restore a backup created with a newer version.
```

Comment:
Optionaly we can make this in bold, because the cynic in me tells that people will stop reading after "can restore backups..." in the previous sentence.

Reply:
I'll bold it and swap the order.